plugin/log/README.md

```diff
@@ -106,3 +107,23 @@ Only log denials for example.org (and below to a file)
     }
 }
 ~~~
+
+Log all queries which were not resolved successfully
+
+~~~ corefile
+. {
+    log . {
+        class denial error
+    }
+}
+~~~
+
+Log all queries on which we did not get errors
+
+~~~ corefile
+. {
+    log . {
+        class denial success
```

Comment:
But also:
~~~
class denial
class succes
~~~
?

Reply:
forgot about it, added it.